packages/app/src/cli/models/app/app.ts

```diff
@@ -276,6 +281,17 @@ export class App implements AppInterface {
       }
     }
 
+    const extensionCollections = this.allExtensions.filter(
+      (ext) => ext.isEditorExtensionCollection,
+    ) as ExtensionInstance<EditorExtensionCollectionType>[]
+
+    if (extensionCollections.length > 0) {
+      const errors = validateExtensionsHandlesInCollection(extensionCollections, this.allExtensions)
+      if (errors) {
+        throw new AbortError('Invalid editor extension collection configuration', errors.join('\n'))
```

Comment:
could we add another `\n` here? I found it a bit harder to see more than one error altogether. Maybe it will be a bit easier to read them if we add an empty line in between?
![Screenshot 2024-04-17 at 15 12 49](https://github.com/Shopify/cli/assets/3033781/7ea5bbd0-1b45-4d62-975c-309e4f1742a9)